plugins/module_utils/dnac.py

```diff
@@ -290,6 +290,7 @@ def get_dnac_params(self, params):
                        "dnac_port": params.get("dnac_port"),
                        "dnac_username": params.get("dnac_username"),
                        "dnac_password": params.get("dnac_password"),
+                       "dnac_version":params.get("dnac_version"),
```

Comment:
Need a space after : 
Can you please check it? Sanities are getting failed.

`"dnac_version": params.get("dnac_version"),`

Reply:
updated the code